src/main.cpp

```
@@ -468,16 +500,37 @@ bool SMPDebug_Execute(
 					reportPath,
 					gearOnly,
 					includeWarnings ? hdt::ValidationReportMode::Full : hdt::ValidationReportMode::ErrorsOnly);
-				auto* console = RE::ConsoleLog::GetSingleton();
+				// Show the report CONTENT in the menu's Output panel, not just the fact it was generated.
+				// Only the tail that fits the menu ring buffer is appended (a full report can be tens of MB),
+				// scanned backwards so the huge file is never split line-by-line from the start. The summary
+				// and file path are echoed after it, so they stay visible at the bottom.
+				if (!reportPath.empty()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)